pkg/drivers/sqlite/sqlite.go

```diff
@@ -35,9 +36,21 @@ var (
 				value BLOB,
 				old_value BLOB
 			)`,
-		`CREATE INDEX IF NOT EXISTS kine_name_id_index ON kine (name,id)`,
-		`CREATE UNIQUE INDEX IF NOT EXISTS kine_prev_revision_name_uindex ON kine (prev_revision, name)`,
 	}
+
+	dropIndices = []string{
```

Comment:
In general, for SQL, I would prefer to be part of the code as it makes things much more clear. Right now, to understand a piece of code, one must go up and down the file to read the SQL that gets executed.

Reply:
I've moved the SQL strings at the top of the file to exactly where they're used. Since each is used only once, it makes sense to move them where they're used. Cheers!